src/components/game/graphics/sections/Queue.js

```diff
@@ -33,4 +53,5 @@ export function updateQueue(pieceProvider) {
   for (const piece of pieceProvider.queue) {
     updatePiece(piece);
   }
+  pieceProvider.yOffset *= 0.8;
```

Comment:
Lift into gameState constant

Reply:
Look for all "*=" mentions and put decay rates into a new constants file called Dynamics.js